src/components/UserProfile/UserProfileEdit/ToggleSwitch/ToggleSwitch.jsx

```diff
@@ -1,8 +1,9 @@
-import React from 'react';
+import { useState } from 'react';
 import style from './ToggleSwitch.module.scss';
 import TriStateToggleSwitch from './TriStateToggleSwitch';
 
-const ToggleSwitch = ({ switchType, state, handleUserProfile, fontSize }) => {
+const ToggleSwitch = ({ switchType, state, handleUserProfile, fontSize, UpdateTeamMembersVisiblity, userId, choice }) => {
+  const[visiblity,setVisiblity]=useState(choice);
```

Comment:
Nitpick but the spacing on this line could probably be improved.
`const [visiblity, setVisiblity] = useState(choice);`